starter-code/routes/index.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+const express = require('express');
+const router = express.Router();
+
+const Movie = require('../models/movies');
+
+/* GET home page. */
+router.get('/', (req, res, next) => {
+  const user = req.session.currentUser;
```

Comment:
You already added app.locals.user in App.js so you don't need it here. You have a user variable available in your templates